src/context.cpp

```diff
@@ -579,6 +599,26 @@ namespace librealsense
             software_sensor::close();
         }
 
+        void add_option( std::shared_ptr< realdds::dds_option > option )
+        {
+            //Convert name to rs2_option type
+            rs2_option opt_type = RS2_OPTION_COUNT;
```

Comment:
`opt_type`? It's not the type of options, but which option it is... maybe `option_id`?

Reply:
Changed